byron/ledger/executable-spec/cs-ledger.cabal

```diff
@@ -80,11 +84,12 @@ test-suite doctests
   if (!flag(development))
     ghc-options:      -Werror
 
-test-suite ledger-delegation-test
+test-suite ledger-rules-test
```

Comment:
I'd suggest a different name here (e.g., `ledger-test`) as this test suite is not about rules, but (predominantly) about testing properties.

Reply:
Noted. This should be merged with your test suite. But I didn't want to name it `sts-properties` to avoid conflict.

Reply:
Oh, now I see that you renamed this to `ledger-test`. Actually I think `ledger-rules-test` or `ledger-sts-tests` describe better what these test do: i.e. not testing the ledger, but testing the rules we wrote for it.